RELEASE.md

```diff
@@ -84,6 +84,38 @@
 [Deepyaman Datta](https://github.com/deepyaman), [Lucas Jamar](https://github.com/lucasjamar), [Simon Brugman](https://github.com/sbrugman)
 
 ## Migration guide from Kedro 0.17.* to 0.18.*
+* If you are using any modular pipelines with parameters, make sure they are declared with the correct namespace. See example below:
+
+For a given pipeline:
+```python
+active_pipeline = pipeline(
+    pipe=pipeline_instance,
```

Comment:
I think we need to show what is being overriden here - we don't really have the 'from and too' currently

```suggestion
    pipe=pipeline(
       [
          node(
             func=some_func,
             inputs=["model_input_table", "params:model_options"]
             outputs=...
          )
       ]
    )
```

Reply:
This doesn't have to be inline, but I think you need to show what parameters are getting overridden

Reply:
Fair point, done.